js/main.js

```diff
@@ -0,0 +1,103 @@
+const addBtn = document.querySelector('.add-btn');
+addBtn.onclick = addBook;
+
+let bookCollection = [];
+let messageOn = false;
+
+// Helper functions
+function renderEmptyMessage() {
+  messageOn = true;
+  const bookList = document.querySelector('.books-display');
+  bookList.innerHTML = '';
+  const message = document.createElement('span');
+  message.innerText = 'Your book collection is empty. Kindly add your favorite books.';
+
+  bookList.appendChild(message);
+  return bookCollection;
+}
+
+function isCollectionEmpty() {
+  if (bookCollection.length === 0) renderEmptyMessage();
+}
+
+// Model
+const savedCollection = JSON.parse(localStorage.getItem('books'));
+
+bookCollection = savedCollection ?? renderEmptyMessage();
+isCollectionEmpty();
+
+if (!messageOn) renderBooks(bookCollection);
+
+// Saves Collection
+function saveCollection() {
+  localStorage.setItem('books', JSON.stringify(bookCollection));
+}
+
+// Creates a Book
+function CreateBook(id, title, author) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+  bookCollection.push(this);
+  saveCollection();
+}
+
+// Deletes a Book
+function removeBook(bookId) {
+  bookCollection = bookCollection.filter(({ id }) => id !== bookId);
+  saveCollection();
+}
```

Comment:
- [ ] Please when I remove any book(`when  I click on the remove button`) from that page,  I deletes all the books, please kindly fix the issue.Please make sure that when you delete a particular, only that book should be deleted and the localStorage updated according accordingly.

Reply:
Thank you for the feedback and note we have made the required changes 😄 Happy Coding @t-yanick